tests/time_tests/c++/pipeline.h

```diff
@@ -0,0 +1,73 @@
+#include <chrono>
+#include <functional>
+
+#include <inference_engine.hpp>
+using namespace InferenceEngine;
+
+
+/**
+ * @brief Class response for encapsulating measure and measurements printing
+ *
+ * Current class measures only in scope of it's lifetime. In this case need
+ * to note that deletion of objects created before class creation may lead
+ * to negative values because of alignment on starting values.
+ * Also deletion of objects created in scope of class lifetime may decrease
+ * values computed on previous measure.
+ */
+class TimePipeline {
```

Comment:
make it global so that some functions like `start/finish` to work with this class

Reply:
think about implementing logic of few timers via one class

Reply:
Done